tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py27, py3
+
+[testenv]
+commands =
+    python setup.py test -q
```

Comment:
As far as I've seen, the `-q` in `python setup.py test -q` doesn't do anything.

Perhaps you mean `python setup.py -q test`, which removes the very verbose and IMHO totally unnecessary setuptools build output that is printed before the tests start running.

Also, we've been switching to `zope-testrunner --test-path=src {posargs}` elsewhere because `setup.py test` is a pain to support.)

Reply:
It's also deprecated and emits a warning on the console when used. It suggests `tox` as a replacement which...ok?

```console
$ python setup.py test
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
...
```